cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/insights-config.js

```diff
@@ -2,56 +2,95 @@
 
 /**
  * Configures Insights
+ * Relies on the objects and events created by SFRA's addToCart handler:
+ * https://github.com/SalesforceCommerceCloud/storefront-reference-architecture/blob/dec9c7c684275127338ac3197dfaf8fe656bb2b7/cartridges/app_storefront_base/cartridge/client/default/js/product/base.js#L624
  * @param {string} appId Application ID
  * @param {string} searchApiKey Search API Key
+ * @param {string} productsIndex Products index name
  */
-function enableInsights(appId, searchApiKey) {
+function enableInsights(appId, searchApiKey, productsIndex) {
+    window.aa('init', {
+        appId,
+        apiKey: searchApiKey,
+    });
 
-    // when on product page
-    document.addEventListener('click', function (event) {
-        if ($(event.target).is('button.add-to-cart')) {
-            // read data from URL
-            var queryID = getUrlParameter('queryID');
-            var objectID = getUrlParameter('objectID');
-            var indexName = getUrlParameter('indexName');
-            if (queryID && objectID && indexName) {
-                window.aa('convertedObjectIDsAfterSearch', {
-                    eventName: 'Product Add to cart',
-                    index: indexName,
-                    queryID: queryID,
-                    objectIDs: [objectID],
-                });
-            }
+    let lastQueryID = null;
+    let lastIndexName = null;
+    let lastPidsOjs = [];
```

Comment:
Oups, a typo indeed!
As for the postinstall, it's true that we commit the files but I was thinking that it can help to not forget to run the command when we upgrade the libraries versions. But it can probably be checked in a different manner, I've reverted for now.